packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -240,7 +240,20 @@ extension TextFromString on String {
   }
 }
 
+final List<Object> _weakValueTests = <Object>[1, 1.0, 'hello', true, false, Object(), <int>[3, 4], DateTime(2023)];
+
 void main() {
```

Comment:
also need a test verifying that objects are no longer kept alive after they are GCed due to using toId.
That test should fail as the code is written right now because _objectToId is keeping all the objects alive.